R/DownloadModule.R

```diff
@@ -243,7 +244,8 @@ reporter_download_inputs <- function(rmd_yaml_args, rmd_output, session) {
             choices = rmd_output,
             selected = rmd_yaml_args$output
           )
-        )
+        ),
+        toc = shiny::checkboxInput(session$ns("toc"), label = "table of content", value = rmd_yaml_args$toc)
```

Comment:
So to be consistent with the SRC. It should say "Include Table of Contents" (note in Eng you need the "s" for Contents)